src/SignalR/run.cmd

```diff
@@ -0,0 +1,2 @@
+@ECHO OFF
```

Comment:
Don't copy the run.{cmd,sh,ps1} scripts here. Instead, please add scripts that look like these:

https://github.com/aspnet/AspNetCore/blob/master/src/Razor/build.cmd
https://github.com/aspnet/AspNetCore/blob/master/src/Razor/build.sh

Reply:
Oh, is that already working?

Reply:
It should work.